docs/source/documentation/team/joining-our-teams.html.md.erb

```diff
@@ -0,0 +1,47 @@
+---
+owner_slack: "#data-platform-notifications"
+title: Joining our teams
+last_reviewed_on: 2023-02-27
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+## Tools
+
+### Communication
+
+We use Google Meet for team calls, and Slack huddles or Google Meet for pairing or direct calls.
+
+Our main chat platform is Slack, and this is our preferred method of communication.
+
+We have three main Slack channels listed below.
+
+#### #data-platform
+
+The [#data-platform](https://mojdt.slack.com/archives/C04BBJR3RPB) channel is the main channel with all members of the Data Plarform.
+
+#### #data-platform-core-infrastructure
+
+The [#data-platform-core-infrastructure](https://mojdt.slack.com/archives/C04BRGD7KFH) channel is our team channel where the Core Infrastructure team discussions about anything ranging from:
```

Comment:
Doesn’t quite read right, could change to “where the Core Infrastructure team have discussions…”